src/pudl/transform/params/ferc1.py

```diff
@@ -3774,6 +3773,101 @@
+                    "accumulated_depreciation_transmission_ending_balance": "transmission_utility_plant_value",
+                    "accumulated_depreciation_distribution_ending_balance": "distribution_utility_plant_value",
+                    "accumulated_depreciation_general_ending_balance": "general_utility_plant_value",
+                    "accumulated_depreciation_regional_transmission_and_market_operation_ending_balance": "regional_transmission_and_market_operation_utility_plant_value",
+                    "accumulated_provision_for_depreciation_of_electric_utility_plant_ending_balance": "ending_balance_functional_utility_plant_value",
+                }
+            },
+            "xbrl": {
+                "columns": {
+                    "entity_id": "utility_id_ferc1_xbrl",
+                    "report_year": "report_year",
+                    "electric_plant_classification_axis": "plant_classification_type",
+                    "utility_type_axis": "utility_type",
+                }
+            },
+        },
+        "categorize_strings": {
+            "utility_type": UTILITY_TYPE_CATEGORIES,
+            "plant_classification_type": ELECTRIC_PLANT_CLASSIFICATION_CATEGORIES,
+        },
+        "align_row_numbers_dbf": {"dbf_table_names": ["f1_accumdepr_prvsn"]},
+        "wide_to_tidy": {
+            "dbf": {
+                "idx_cols": [
+                    "utility_id_ferc1",
+                    "report_year",
+                    "plant_functional_type",
+                    "record_id",
+                ],
+                "value_types": ["utility_plant_value"],
+                "expected_drop_cols": 2,
+                "stacked_column_name": "plant_classification_type",
+            },
+            "xbrl": {
+                "idx_cols": [
+                    "entity_id",
+                    "report_year",
+                    "electric_plant_classification_axis",
+                    "utility_type_axis",
+                ],
+                "value_types": ["utility_plant_value"],
+                "expected_drop_cols": 10,
+                "stacked_column_name": "plant_functional_type",
+            },
+        },
+        "merge_xbrl_metadata": {
+            "rename_columns": {"xbrl_factoid": "plant_functional_type"},
+            "on": "plant_functional_type",
+        },
+        "select_dbf_rows_by_category": {
+            "column_name": "plant_functional_type",
+            "select_by_xbrl_categories": False,
+            "additional_categories": [
+                "distribution",
+                "general",
+                "hydraulic_production_conventional",
+                "hydraulic_production_pumped_storage",
+                "nuclear_production",
+                "other_production",
+                "regional_transmission_and_market_operation",
+                "steam_production",
+                "transmission",
+                "ending_balance_functional",
+            ],
```

Comment:
if you convert the `ending_balance_functional` to `total` and then add the `total` into `PLANT_FUNCTIONAL_CATEGORIES`, could you use the keys from `PLANT_FUNCTIONAL_CATEGORIES` instead of writing these out here?